pkg/apis/ceph.rook.io/v1/webhook.go

```diff
@@ -0,0 +1,89 @@
+package v1
```

Comment:
How about separate source files for different CRDs? For example, validate_cluster.go and validate_pool.go